### DISCUSSION TITLE
LosslessCut Cuts Frame Rates in Half

### DISCUSSION THREAD

**Discussion by zombiegak** [mifi/lossless-cut#1595] [General]
I have an old 2004 Kodak camera that shot 480p video at about 12.5 fps. This is back in the day when a 500meg SD card cost $60. 

I still have plenty of old files that need to be merged, deblocked and compressed. I've used EasiestSoft merger which works well but doesn't work with other camera or smartphone video. So I hoped LosslessCut could do it all. While LC has worked fantastically on videos from other sources I noticed on my first attempt with these old Kodak files that LosslessCut cut the frame rate in half. If anyone's interested in why LC is doing this I can upload some of these old videos to be tested to see what's going on.

**Comment by mifi** [comment]
Does this also happen when cutting only? (as opposed to cut+merge). if you could share a small file that would be useful yes
